pom.xml

```diff
@@ -13,5 +13,12 @@
         <maven.compiler.target>23</maven.compiler.target>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     </properties>
+    <dependencies>
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter-engine</artifactId>
+            <version>5.11.2</version>
+        </dependency>
+    </dependencies>
```

Comment:
Feedback pom.xml:

Let op, voor je eindopdracht mag er geen SNAPSHOT in je <version> staan (regel 9). 

En voor je eindopdracht zou ik nog even bij de docenten checken of je versie 23 mag gebruiken (regel 12-13). Dat is geen long term support versie.